core/build-image.sh

```diff
@@ -30,22 +32,35 @@ buildah run nodebuilder-core sh -c "cd /usr/src/core/ui       && npm install &&
 
 echo "Download Logcli..."
 logcli_tmp_dir=$(mktemp -d)
-wget https://github.com/grafana/loki/releases/download/v2.2.1/logcli-linux-amd64.zip -P ${logcli_tmp_dir}
-unzip ${logcli_tmp_dir}/logcli-linux-amd64.zip -d ${logcli_tmp_dir}
+cleanup_list+=("${logcli_tmp_dir}")
+wget https://github.com/grafana/loki/releases/download/v2.2.1/logcli-linux-amd64.zip -P "${logcli_tmp_dir}"
+unzip "${logcli_tmp_dir}/logcli-linux-amd64.zip" -d "${logcli_tmp_dir}"
 
 echo "Building the Core image..."
 container=$(buildah from scratch)
 reponame="core"
 buildah add "${container}" imageroot /
-buildah add "${container}" ${logcli_tmp_dir}/logcli-linux-amd64 /usr/local/sbin/logcli
-rm -r ${logcli_tmp_dir}
-trap "rm -rf ${logcli_tmp_dir}" EXIT
+buildah add "${container}" "${logcli_tmp_dir}/logcli-linux-amd64" /usr/local/sbin/logcli
 buildah add "${container}" agent/agent /usr/local/bin/agent
 buildah add "${container}" api-server/api-server /usr/local/bin/api-server
 buildah add "${container}" ui/dist /var/lib/nethserver/cluster/ui
 buildah config \
     --label='org.opencontainers.image.source=https://github.com/NethServer/ns8-scratchpad' \
+    --label="org.nethserver.images=ghcr.io/nethserver/redis:${IMAGETAG:-latest}" \
     --entrypoint=/ "${container}"
+cleanup_list+=("coreimage.lst")
+if [[ "${EUID}" == "0" ]]; then
+    cmd_buildmeta="env cmnt=$(buildah mount ${container})"
+else
+    cmd_buildmeta="buildah unshare --mount cmnt=${container}"
+fi
+${cmd_buildmeta} /bin/bash -s <<'EOSHARE'
+printf "CORE_IMAGE=ghcr.io/nethserver/core:%s\n" "${IMAGETAG:-latest}" >> "${cmnt}/etc/nethserver/core.env"
+printf "REDIS_IMAGE=ghcr.io/nethserver/redis:%s\n" "${IMAGETAG:-latest}" >> "${cmnt}/etc/nethserver/core.env"
+find "${cmnt}" -printf "%P\n" | sort > coreimage.lst
+EOSHARE
+buildah unmount "${container}"
+buildah add "${container}" coreimage.lst /var/lib/nethserver/node/state/coreimage.lst
```

Comment:
Just for the record, this is how to mount the image during the build to generate its file list